docs/src/examples/components/Dropdown/Types/index.tsx

```diff
@@ -9,6 +9,16 @@ const Types = () => (
       description="A dropdown with single selection."
       examplePath="components/Dropdown/Types/DropdownExampleSingleSelection"
     />
+    <ComponentExample
+      title="Single Search"
+      description="A dropdown with single selection and search."
+      examplePath="components/Dropdown/Types/DropdownExampleSingleSearch"
+    />
+    <ComponentExample
+      title="Multiple Selection"
+      description="A dropdown with multiple selection."
+      examplePath="components/Dropdown/Types/DropdownExampleMultipleSelection"
+    />
```

Comment:
I will be glad if we will couple these example somehow, i.e. one section for the prop, but multiple examples:

![image](https://user-images.githubusercontent.com/14183168/52262213-8f41c080-2934-11e9-8f7c-a898d323614d.png)

https://react.semantic-ui.com/modules/dropdown/#types-search-selection

Reply:
Not sure I follow. In this case we have 2 boolean props, `search` and `multiple` and we need examples for all permutations of these 2, resulting in 4 examples:

```jsx
<Dropdown ../> {/* single selection */}
<Dropdown ../> {/* single selection */}
<Dropdown ../> {/* single selection */}

Reply:
I like it, pushed a commit to implement it; this is how it looks right now:
(the multiple flavor was separated and currently implemented in #845 )
![screenshot 2019-02-06 at 15 18 05](https://user-images.githubusercontent.com/5442794/52347550-7b748800-2a22-11e9-9eb2-9bc891d6ef65.png)

What do you think @layershifter ?

Reply:
Looks pretty clear to me ❤️

Reply:
great, resolving